sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -1,27 +1,38 @@
 import {
-  COESummaryDTO,
+  COESummaryAndCount,
   ApplicationDetailsForCOEDTO,
   COEDeniedReasonDto,
   DenyConfirmationOfEnrollment,
   EnrollmentPeriod,
+  PaginationOptions,
+  FieldSortOrder,
+  DataTableSortOrder,
+  PaginationParams,
 } from "@/types";
 import HttpBaseClient from "./common/HttpBaseClient";
 
 export class ConfirmationOfEnrollmentApi extends HttpBaseClient {
   public async getCOESummary(
     locationId: number,
     enrollmentPeriod: EnrollmentPeriod,
-  ): Promise<COESummaryDTO[]> {
-    try {
-      const response = await this.apiClient.get(
-        `institution/location/${locationId}/confirmation-of-enrollment/enrollmentPeriod/${enrollmentPeriod}`,
-        this.addAuthHeader(),
-      );
-      return response.data;
-    } catch (error) {
-      this.handleRequestError(error);
-      throw error;
+    paginationOptions: PaginationOptions,
+  ): Promise<COESummaryAndCount> {
+    const apiSortOrder =
+      paginationOptions.sortOrder === DataTableSortOrder.DESC
+        ? FieldSortOrder.DESC
+        : FieldSortOrder.ASC;
+    let URL = `institution/location/${locationId}/confirmation-of-enrollment/enrollmentPeriod/${enrollmentPeriod}`;
+    /**Adding Sort params. There is always a default sortField and sortOrder. */
+    URL = `${URL}?${PaginationParams.SortField}=${paginationOptions.sortField}&${PaginationParams.SortOrder}=${apiSortOrder}`;
```

Comment:
there is function to add sort option to the URL, 
![image](https://user-images.githubusercontent.com/77353155/153684782-1752dda6-d550-4b57-9c3d-abdc9e2fbe67.png)
consumer, 
![image](https://user-images.githubusercontent.com/77353155/153684799-a99ba4c5-ae24-40b7-9ed3-a266e06b0787.png)

Reply:
SortOptions Utils have been leveraged.